15/content_sets.yml

```diff
@@ -0,0 +1,10 @@
+# This is a file defining which content sets are needed to update content in 
```

Comment:
This file is not needed at all. Please remove it.

Reply:
Yes, content_sets.yml is not synced at all. RHCWT config file for RHEL8 and row 33.

Reply:
Thanks for confirming, removed.